interpreter/src/main/scala/pl/touk/nussknacker/engine/compile/ComponentExecutorFactory.scala

```diff
@@ -34,9 +35,9 @@ class FactoryComponentInvoker(expressionEvaluator: ExpressionEvaluator) extends
     }
   }
 
-  private def doInvokeCreateMethod[T](
+  private def doCreateComponentExecutor[T](
```

Comment:
```suggestion
  private def doCreateComponentExecutor[ComponentExecutor](
```
for consistencies sake